src/promptflow/promptflow/_cli/_params.py

```diff
@@ -291,3 +293,49 @@ def add_param_config(parser):
 
 
 logging_params = [add_param_verbose, add_param_debug]
+
+
+def add_param_archived_only(parser):
+    parser.add_argument(
+        "--archived-only",
+        action="store_true",
+        help="Only list archived records.",
+    )
+
+
+def add_param_include_archived(parser):
+    parser.add_argument(
+        "--include-archived",
+        action="store_true",
+        help="List both archived records and active records.",
+    )
+
+
+def add_param_output_format(parser):
+    parser.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        default=CLIListOutputFormat.JSON,
+        help="Output format, accepted values are 'json' and 'table'. Default is 'json'.",
+        choices=[CLIListOutputFormat.TABLE, CLIListOutputFormat.JSON],
+    )
+
+
+def add_param_owned_only(parser):
+    parser.add_argument(
+        "--owned-only",
```

Comment:
Maybe we should do the opsite? default only show your own, and need a swtich to show all? 
If we keep this, maybe we can name the  parameter follow our UI?  --only-my-flows
![image](https://github.com/microsoft/promptflow/assets/47586720/3f23f097-b12a-4ec8-bd4d-341eac73e8de)

Reply:
changed to `--include-others`